lib/config.js

```diff
@@ -15,15 +15,15 @@ module.exports = {
     go: 15000,
     haskell: 15000,
     sql: 14000,
-    java: 20000
+    java: 20000,
+    kotlin: 20000,
```

Comment:
I know Kotlin takes longer to compile, but I'm not sure how much longer. It really depends on the complexity of the target. On Travis, minimum challenges finish in ~5 sec with Gradle daemon running. I don't know how long it'll take on AWS t2.small with other containers.
We can adjust later if this is not enough. We can also adjust by dependencies used like Java runner does. This is just the default.